env/postgres.env.example

```diff
@@ -1,5 +1,5 @@
-MAGPIE_POSTGRES_USERNAME=postgres
+MAGPIE_POSTGRES_USERNAME=magpie
 MAGPIE_POSTGRES_PASSWORD=qwerty
-MAGPIE_POSTGRES_HOST=postgres
+MAGPIE_POSTGRES_HOST=localhost
```

Comment:
I would had a note because this is probably one of the most confusing thing usually. 
If running docker magpie and using a local postgres, localhost is right. 
But inside a docker-compose stack, the name of the linked postgres image must be used to find the network.